server/base/src/main/java/org/apache/accumulo/server/conf/store/PropStoreKey.java

```diff
@@ -89,25 +89,44 @@ protected PropStoreKey(final InstanceId instanceId, final String path, final ID_
    */
   public static @Nullable PropStoreKey<?> fromPath(final String path) {
     String[] tokens = path.split("/");
+    if (tokens.length < 1) {
+      return null;
+    }
 
     InstanceId instanceId;
     try {
       instanceId = InstanceId.of(tokens[IID_TOKEN_POSITION]);
+
```

Comment:
Alternatively, you could remove the try/catch here and check that tokens.length equals EXPECTED_CONFIG_LEN or EXPECTED_SYS_CONFIG_LEN.

Reply:
addressed in b9ef24d03b